.github/workflows/compile-queries.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         with:
           channel: 'release'
       - name: check formatting
-        run: codeql query format */ql/{src,lib,test}/**/*.{qll,ql} --check-only
+        run: codeql query format */ql/**/*.{qll,ql} --check-only
```

Comment:
So, this expands the format check to _every_ ql and qll file in the repo. I think that's fine.